torch/_torch_docs.py

```diff
@@ -6090,7 +6098,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.randint_like,
            """
-randint_like(input, low=0, high, dtype=None, layout=torch.strided, device=None, requires_grad=False, \
+randint_like(input, low=0, high, \\*, dtype=None, layout=torch.strided, device=None, requires_grad=False, \
```

Comment:
What's up with the "\\*" here?

Reply:
Some docs use `\\*` and some use `*`, I don't think they are different. I compiled and looked at the generated page, and I can not see the difference either. Here I used `\\*` simply because there is a similar operator also uses `\\*`.

Reply:
Eventually we should fix that inconsistency to avoid further confusion. This PR looks great, though.